### PR TITLE
Works that are inquireable and acquireable should behave correctly

### PR DESCRIPTION
**Pull request by sweir27**
The goal of this PR is to unblock https://github.com/artsy/metaphysics/pull/1249.

Until we launch the BNMO feature, we want (on desktop + mobile web):
- Auction-partner artworks that are `inquireable` _and_ `acquireable` should show the "Buy" button only
- Gallery-partner artworks that are `inquireable` _and_ `acquireable` should show the "Inquire" button _unless_ you have the lab feature, in which case you should see the "Buy" button
- Gallery-partner artworks that are `inquireable` and _not_ `acquireable` should show the "Inquire" button
- Works in an auction that are also `acquireable` should show the bid form + the "Buy" button

When we launch the BNMO flow, we will update this to be simpler:
- Works that are `inquireable` and `acquireable` should _just_ show the buy button (across the board)

### CODE REVIEW

File: src/desktop/apps/artwork/components/commercial/templates/inquire.jade [hunk 1]
```diff
@@ -1,4 +1,8 @@
-if artwork.is_inquireable
+- var auctionPartner = artwork.partner.type === 'Auction' || artwork.partner.type === 'Auction House'
+- var newBuyNowFlow = user && user.hasLabFeature("New Buy Now Flow")
+
+//- TODO BNMO: Update to hide if the artwork is acquireable
+if artwork.is_inquireable && !((artwork.is_acquireable && auctionPartner) || newBuyNowFlow)
```
Comment (sweir27) [inline src/desktop/apps/artwork/components/commercial/templates/inquire.jade]: This is the component that shows the "inquire" button on the sidebar of the artwork page.

You should see the inquire button if the artwork is `inquireable` _and_ you're not an auction partner with an `acquireable` work, and you aren't enabled for the lab feature.

Artworks that are part of auctions in any way don't show this component at all.

File: src/desktop/apps/artwork/components/partner/index.jade [hunk 2]
```diff
@@ -23,6 +23,7 @@ if helpers.partner.isDisplayable(artwork.partner)
           = profile.bio
 
       .artwork-partner__metadata__actions
+         //- TODO BNMO: Update to hide if the artwork is acquireable
```
Comment (sweir27) [inline src/desktop/apps/artwork/components/partner/index.jade]: This component is a little partner info widget on the artwork view:
image

It only shows if the artwork's partner is a [gallery or an institution](https://github.com/artsy/force/blob/master/src/desktop/apps/artwork/components/partner/helpers.coffee#L5).

So, when we eventually want to suppress inquiry buttons for `acquireable` artworks, we'll have to update this. For now it's okay.

File: src/desktop/apps/artwork/test/fixtures/acquireable_artwork.json [hunk 3]
```diff
@@ -4,7 +4,7 @@
       "_id": "56e86588b202a366da000571",
       "id": "yee-wong-exploding-powder-movement-blue-and-pink",
       "is_acquireable": true,
-      "is_inquireable": false,
+      "is_inquireable": true,
```
Comment (sweir27) [inline src/desktop/apps/artwork/test/fixtures/acquireable_artwork.json]: Artworks that are `acquireable` tend to also be `inquireable`. This makes our tests a little more robust.

File: src/desktop/components/artwork_metadata_stub/templates/contact.jade [hunk 4]
```diff
@@ -20,7 +22,8 @@ if artwork.sale && artwork.sale.is_auction
           | #{sa.opening_bid.display}
       else if artwork.sale.is_closed
         | Auction closed
-else if artwork.is_inquireable
+else if artwork.is_inquireable && !(artwork.is_acquireable && auctionPartner)
```
Comment (sweir27) [inline src/desktop/components/artwork_metadata_stub/templates/contact.jade]: This is the byline on the artwork brick.

If the artwork is `inquireable`,  we show "Contact Gallery". Here, we update it to _not_ show that CTA if the artwork is `acquireable` and belongs to an auction partner.

File: src/mobile/apps/artwork/components/meta_data/templates/inquiry.jade [hunk 5]
```diff
@@ -10,7 +10,8 @@ if buyable && (isAuction || newBuyNowFlow || isAuctionPartner)
     else
       | Buy
 
-unless artwork.auction
+ //- TODO BNMO: Update to hide inquiry flow if the artwork is acquireable
+unless (isAuction || newBuyNowFlow || isAuctionPartner)
```
Comment (sweir27) [inline src/mobile/apps/artwork/components/meta_data/templates/inquiry.jade]: This is the mobile-web version of the commercial sidebar.

File: src/desktop/components/artwork_metadata_stub/test/index.js [hunk 6]
```diff
@@ -25,6 +25,47 @@ describe('metadata template', () => {
...
+      artwork = {
+        is_acquireable: true,
+        is_inquireable: true,
+        date: '2007',
+        title: 'My Artwork',
+        partner: { type: 'Auction House', href: '/partner/auction-partner' },
+      }
+    })
+
+    it('shows an inquiry CTA if the artwork is inquireable', () => {
```
Comment (zephraph) [inline src/desktop/components/artwork_metadata_stub/test/index.js]: Should say that it does not show the inquiry CTA

**Pull request merged**